docs/usage/basics/shoot_access.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Accessing Shoot Clusters
```

Comment:
It is not clear at all for me when you decide to set a title or not? Was it documented somewhere? I recall that hugo (?) generates a title based on the title in the markdown. However things here seems to be straightforward so why we add title here but for example we don't add title to the `service-account-manager.md` ?

Reply:
I don't recall whether we have a developer guide for docs: for example, use relative links, etc. If we have such guide, we can enhance it with the new things from this PR: for example, add description to your docs to improve the search quality in the website.